src/main/kotlin/org/rust/lang/core/resolve/Processors.kt

```diff
@@ -52,6 +55,8 @@ fun collectResolveVariants(referenceName: String, f: (RsResolveProcessor) -> Uni
 fun collectCompletionVariants(f: (RsResolveProcessor) -> Unit): Array<LookupElement> {
     val result = mutableListOf<LookupElement>()
     f { e ->
+        if((e.element as? RsFunction?)?.isTest ?: false)
```

Comment:
A space is missing after the `if` ;) 

Also we usually format short ifs on a single line: `if (cond) return false`, or add `{}`.